.github/workflows/test.yaml

```diff
@@ -1,36 +1,40 @@
 name: Test
 
-on: [push, pull_request]
+on: [pull_request]
 
 jobs:
   test:
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
-        # with:
-        #   python-version: '3.8'
+
       - uses: conda-incubator/setup-miniconda@v2
         with:
           auto-update-conda: true
           python-version: 3.8
 
+      - shell: bash -l {0}
+        run: |
+          which pip
+          pip --version
+          which python
+          python --version
+          which conda
+          conda --version
+
+
       - shell: bash -l {0}
         run: |
           pip install mypy
           pip install wheel
 
-      - name: Python Poetry Action
```

Comment:
No more poetry? How do we install dependencies and create a package?

Reply:
I was thinking basic `pip install -r ` and to build the package `python setup.py build`